src/usr/local/www/system_advanced_network.php

```diff
@@ -84,6 +85,17 @@
 			unset($config['system']['prefer_ipv4']);
 		}
 
+		if (!empty($_POST['global-v6duid'])) {
+			strtolower(str_replace("-", ":", $_POST['global-v6duid']));
+			if (!is_duid($_POST['global-v6duid']) || strlen($_POST['global-v6duid']) != 47) {
```

Comment:
You don't need to check strlen(), is_duid() will return false if length is not correct